### DISCUSSION TITLE
Anyway to discover FinderID now?

### DISCUSSION THREAD

**Discussion by tryin100** [GerryDazoo/Slinger#312] [Q&A] [answered]
Now that Slingbox has been shut down is there anyway to discover the Finder ID of the slinbox 500s that I have - or, are they now just landfill?
thanks

**Comment by GerryDazoo** [comment, marked as answer]
Run the server without an IP address configured for the slingbox and it will scan your LAN for boes and report IP, port and finderid. Although you don't need the finderid to make slinger work.

**Reply by tryin100** [reply]
Thank you!
It was my understanding from reading other posts that the Finder ID was needed for remote access and viewing.
If that was a misunderstanding on my part please let me know.

**Comment by GerryDazoo** [comment]
It's only needed if you don't have a static WAN IP and you don't already have a DDNS and you want to use my little web service,
